README.md

```diff
@@ -1,19 +1,27 @@
 # xmlvirshparser
 
-Python script to parse the output of "virsh dumpxml" command captured with sosreport
+Python script to parse the output of `virsh dumpxml` command captured with sosreport
 
 # Installation
 
 ## Install using pip
 
-	sudo pip install xmlvirshparser
+        virtualenv xmlvirshparser
```

Comment:
Can we also add step to create python virtual environment for both versions?

```
# Python 2.7
# Create virtual environment
virtualenv ~/myvirtualenvs/xmlvirshparser
# Activate virtual environment
source ~/myvirtualenvs/xmlvirshparser/bin/activate

# Python 3.5
# Create virtual environment
pyvenv ~/myvirtualenvs/xmlvirshparser
# Activate virtual environment
source ~/myvirtualenvs/xmlvirshparser/bin/activate
```

Alternatively use [setv](https://github.com/psachin/setv) to manage Python virtual environment.